quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/services/resources/ConstantsDebugHostname.java

```diff
@@ -26,34 +30,16 @@ public class ConstantsDebugHostname {
             "X-Forwarded-For"
     };
 
-    public static final String[] RELEVANT_OPTIONS = {
-            "hostname",
-            "hostname-url",
-            "hostname-admin",
-            "hostname-admin-url",
-            "hostname-strict",
-            "hostname-strict-backchannel",
-            "hostname-strict-https",
-            "hostname-path",
-            "hostname-port",
-            "proxy",
-            "proxy-headers",
-            "http-enabled",
-            "http-relative-path",
-            "http-port",
-            "https-port"
-    };
-
     public static final String[] RELEVANT_OPTIONS_V2 = {
```

Comment:
Nice idea.

nitpick: What about using directly `List<Option<?>>` with the List.of() ? As we will always expect some KC options, right? Might be even clearer.

Reply:
ok, agree